src/io/ExportVTK.hpp

```diff
@@ -3,57 +3,88 @@
 #include <Eigen/Core>
 #include <iosfwd>
 #include <string>
+#include <vector>
 #include "Export.hpp"
 #include "logging/Logger.hpp"
 
 namespace precice {
 namespace mesh {
 class Mesh;
-}
+class Edge;
+class Triangle;
+} // namespace mesh
 } // namespace precice
 
 namespace precice {
 namespace io {
 
-/// Writes polygonal, or triangle meshes to vtk files.
+/// Writes meshes to xml-vtk files. Only for parallel usage. Serial usage (coupling mode) should still use ExportVTK
```

Comment:
Could you explain this comment more in detail? Why distinguish these cases here? The class is actually called `ExportVTK`.

Reply:
That's a copy-paste artefact. Thanks for spotting it!